.github/workflows/tests.yml

```diff
@@ -50,7 +54,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install -r requirements.txt
+        python -m pip install -r requirements.txt -Iv apache-airflow==${{ matrix.airflow-version }}
```

Comment:
Here is what `pip help` says about `-I`: "This can break your system if the existing package is of a different version or was installed with a different package manager!"

Does it have any implications? Smth like installing dependencies of incorrect versions?